test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -71,3 +71,41 @@ Import-Module Random
     }
 
 }
+
+Describe 'Module reloading with Class definition' -Tags "CI" {
+
+    BeforeAll {
+        Set-Content -Path TestDrive:\TestModule.psm1 -Value @'
+$passedArgs = $args
+class Root { $passedIn = $passedArgs }
+function Get-PassedArgsRoot { [Root]::new().passedIn }
+function Get-PassedArgsNoRoot { $passedArgs }
+'@
+    }
+
+    AfterEach {
+        Remove-Module TestModule -Force -ErrorAction SilentlyContinue

```

Comment:
On a second thought, I'm not sure if it's really worthy to run each `It` in an individual runspace. Here are my arguments:
1. it's slow to run each `It` in a separate runspace.
2. module reloading is supposed to work in this way no matter the runspace is clean or not. Running the tests in a "not-so-clean" runspace provide extra benefit to see if it doesn't work properly in such case.